MapboxDirections/MBDirections.swift

```diff
@@ -168,6 +168,11 @@ open class Directions: NSObject {
         let url = self.url(forCalculating: options)
         let task = dataTask(with: url, completionHandler: { (json) in
             let response = options.response(from: json)
+            if let routes = response.1 {
+                for route in routes {
+                    route.accessToken = self.accessToken
```

Comment:
We need to pair this with `apiEndpoint`, since an access token only makes sense for a particular endpoint (think api.mapbox.com versus api.mapbox.cn).